security/advancedtls/examples/credential_reloading_from_files/server/main.go

```diff
@@ -0,0 +1,107 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// The server demonstrates how to consume and validate OAuth2 tokens provided by
```

Comment:
Is this true? I don't see anything about OAuth2 tokens being consumed or validated. Same for the package level comment on the client side.

Reply:
Ah sorry, this is something I pasted from another example. I've updated the text accordingly.